src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  width: 96px;
+  height: 16px;
+  margin: 0;
+  padding: 0;
+
+  &_star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 16px;
+    width: 16px;
+  }
+
+  &_star:nth-child(-n + 4) {
```

Comment:
Use all the modifiers from the Stars task. It should be possible to set any amount of stars